tests/groupby_test.py

```diff
@@ -187,6 +187,11 @@ def test_boolean_arrays(self):
         self.assertEqual(ct[1], true_ct)
         self.assertListEqual(k.to_ndarray().tolist(), [False, True])
 
+        k, ct = g.size()
```

Comment:
Can we add a comment that size is being added and tested for the issue number? Its not completely necessary, just helps keep track of why things were added and when the testing was added.

Reply:
Done